celery/tests/tasks/test_tasks.py

```diff
@@ -124,7 +131,6 @@ class MyCustomException(Exception):
 
 
 class test_task_retries(TasksCase):
-
```

Comment:
there *should* be space between methods in a class, that includes the first method :)